python/ray/util/annotations.py

```diff
@@ -0,0 +1,82 @@
+def PublicAPI(*args, **kwargs):
+    """Annotation for documenting public APIs.
+
+    Public APIs are classes and methods exposed to end users of Ray. You
+    can expect these APIs to remain backwards compatible even across major Ray
```

Comment:
"major" refers to going from 1.4 -> 1.5, right? not from 1.5 -> 2.0.

Reply:
Major is 1.0 to 2.0. https://semver.org/

Reply:
Why would we guarantee backward compatibility from 1.0 to 2.0? Don't people already except backward incompatibility in that situation?

Reply:
+1 to @robertnishihara . Major versions are the right timing to introduce necessary incompatibilities

Reply:
Er right, this is just a typo. I always forget we release minor versions since the major version basically never changes.